Sources/GIF/code/GIFEncoder.swift

```diff
@@ -2,6 +2,8 @@ import Foundation
 import Logging
 import CairoGraphics
 import Utils
+import QuartzCore
+import System
```

Comment:
I think we need to be careful not to introduce dependencies on Apple frameworks that break the Linux build here (which is one of the core goals of this project). GCD is available on Linux too, so we should be able to get by with

```suggestion
import Dispatch
```

Reply:
I used QuartzCore for `CACurrentMediaTime()` to test whether optimizations helped or harmed performance. I couldn't detect a measurable difference in the optimization that fuses loop iterations for bit-by-bit encoding. Theoretically it would improve performance, not harm performance, by reducing the total number of instructions issued.